content/pt/docs/concepts/observability-primer.md

```diff
@@ -0,0 +1,156 @@
+
+**Telemetria** refere-se aos dados emitidos por um sistema e seu comportamento.
+Esses dados podem vir na forma de [rastros](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) e
+[logs](/docs/concepts/signals/logs/).
+
+**Confiabilidade** responde a pergunta: "O serviço está fazendo o que os
+usuários esperam que ele faça?" Um sistema pode estar ativo 100% do tempo, mas,
+se quando um usuário clica em "Adicionar ao Carrinho" para adicionar um par de
+sapatos pretos ao carrinho de compras, o sistema nem sempre adiciona sapatos
+pretos, então o sistema pode ser **não** confiável.
+
+**Métricas** são dados numéricos sobre sua infraestrutura ou aplicação agregados
+ao longo de um período de tempo. Exemplos incluem: taxa de erro do sistema,
+utilização da CPU e taxa de requisições para um determinado serviço. Para mais
+informações sobre métricas e como elas se relacionam com o OpenTelemetry,
+consulte [Métricas](/docs/concepts/signals/metrics/).
+
+**SLI**, ou _Service Level Indicator_, representa uma medida do comportamento de
+um serviço. Um bom SLI mede seu serviço do ponto de vista dos seus usuários. Um
+exemplo de SLI pode ser a velocidade de carregamento de uma página web.
+
+**SLO**, ou _Service Level Objective_, representa a forma como confiabilidade é
+comunicada para uma organização/outras equipes. Isso é feito associando um ou
+mais SLIs ao valor de negócio.
+
+## Compreendendo rastreamento distribuído
+
+O rastreamento distribuído permite que você observe as requisições que se
+propagam por sistemas complexos e distribuídos. O rastreamento distribuído
+melhora a visibilidade da saúde da sua aplicação ou sistema, permitindo que você
+faça depurações de comportamentos difíceis de reproduzir localmente. É essencial
+para sistemas distribuídos, que costumam ter problemas não determinísticos ou
+que são muito complicados de reproduzir localmente.
+
+Para entender o rastreamento distribuído, você precisa compreender a função de
+cada um de seus componentes: logs, trechos e rastros.
+
+### Logs
+
+Um **log** é uma mensagem com registro de data e hora emitida por serviços ou
+outros componentes. Ao contrário dos [rastros](#distributed-traces), eles não
+são necessariamente associados a uma requisição de usuário ou transação
+específica. Você pode encontrar logs em praticamente todos os softwares. Logs
+foram muito utilizados no passado tanto por equipes de desenvolvimento quanto de
+operação para entender o comportamento de um sistema.
+
+Exemplo de log:
+
+```text
+I, [2021-02-23T13:26:23.505892 #22473]  INFO -- : [6459ffe1-ea53-4044-aaa3-bf902868f730] Started GET "/" for ::1 at 2021-02-23 13:26:23 -0800
+```
+
+Logs não são suficientes para rastrear a execução do código, pois geralmente não
+apresentam informações contextuais, como de onde foram chamados.
+
+Eles se tornam muito mais úteis quando incluídos como parte de um
+[trecho](#spans) ou quando são correlacionados com um rastro e um trecho.
```

Comment:
```suggestion
[trecho](#trechos) ou quando são correlacionados com um rastro e um trecho.
```

Reply:
as of today we keep the anchors untranslated, so the Spans section should have

```
# Trechos {#spans}
```

as title

Reply:
Ohh TIL, thanks!

Should we update all pages that have already been translated so every title matches the original anchor?

Reply:
@lgfa29 good question, in this example: 
Would it look like this?
pt/docs/concepts/observability-primer.md
`## O que é Observabilidade? {#what-is-observability}`